pos_order_to_sale_order/static/src/xml/CreateOrderPopup.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates id="template" xml:space="preserve">
+
+    <t t-name="pos_order_to_sale_order.CreateOrderPopup" owl="1">
+        <div class="popup popup-create-sale-order">
+
+            <div class="modal-header">
+                <h3 class="modal-title w-100">Create Order</h3>
```

Comment:
```suggestion
                <h3 class="modal-title w-100">Create Sale Order</h3>
```

Reply:
why this change?

Reply:
you mean: why I change this html? Because, new pop up in pos 17.0 is different structure

Reply:
no I mean, why the change from `Create Sale Order` to `Create Order` (confusing)

Reply:
this is header
![image](https://github.com/user-attachments/assets/0f47cdd5-57fa-444f-9f04-863a3e9a39e2)

Reply:
ohh, sorry, I read wrong